modules/network/src/main/java/org/apache/ignite/network/scalecube/ScaleCubeClusterServiceFactory.java

```diff
@@ -131,7 +138,7 @@ public void onMembershipEvent(MembershipEvent event) {
                                 topologyService.onMembershipEvent(event);
                             }
                         })
-                        .config(opts -> opts.memberAlias(consistentId))
+                        .config(opts -> opts.memberAlias(consistentId).metadata(new NodeMetadata(restConfiguration.port().value())))
```

Comment:
No, the rest configuration is taken from a file but there is a `portRange` config that could be used if the original rest.port is not available. So the REST server can be started on another port, look at `RestComponent.start()`.

Reply:
done